UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExcangeSetFssRedirectResponse.cs

```diff
@@ -0,0 +1,60 @@
+﻿using NUnit.Framework;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.FunctionalTests.Helper;
+using System.Collections.Generic;
+
+namespace UKHO.ExchangeSetService.API.FunctionalTests.FunctionalTests
+{
+    class ExcangeSetFssRedirectResponse
```

Comment:
Kindly correct typo of class name and file ExcangeSetFssRedirectResponse to ExchangeSetFssRedirectResponse.cs. Also remove line breaks at 17,18

Reply:
Thanks for highlighting this, did changes accordingly.